### ISSUE TITLE
Error loading plugin if game has Player class defined

### ISSUE THREAD

**Issue by sutyrin** [GamePushService/GamePush-Godot-plugin#4]
![image](https://github.com/user-attachments/assets/f8f71a5d-cbe8-4af6-b010-206546678c23)

This may be fixed by renaming local class to something else, like this: 

![image](https://github.com/user-attachments/assets/48212044-4952-4643-86f8-88d4e92a111b)

**Comment by talkafk** [comment]
Unfortunately, in Godot, the names of internal classes conflict with global names. While I hope this will be fixed, the current solution is to rename all internal plugin classes in the plugin. For example, renaming `Player` to `GPPlayer` to avoid conflicts. I suppose the classes will be renamed in upcoming versions.

**Comment by sutyrin** [comment]
Yeah, I agree. It seems that plugin classes should have more specific names, rather than users'.
